content/_index.md

```diff
@@ -7,10 +7,26 @@ BookToC: false
 
 # The Linux Userspace API (UAPI) Group
 
-The userspace API ("uapi") group is a community for people with an interest in innovating how we build, deploy, and run modern Linux operating systems.
+The userspace API ("uapi") group is a cross-distro community for people with an interest in innovating how we build, deploy, run, and safely update modern Linux operating systems.
 It serves as a central gathering place for specs, documentation, and ideas.
 
-Follow [@uapi-group](https://twitter.com/uapi_group) on twitter to catch the latest news!
+In scope of the group are secure boot mechanisms, kernel / disk image formats, TPM usage, safe and secure updates, and related topics.
+The group also cares about interop of image-based and/or immutable Linux distributions.
```

Comment:
I would delete this section as it restricts us to specific topics. I don't think we can do that currently as we're still exploring the problem space.

Reply:
I'll phrase it more exploratory then. Added this to provide context on what's on-topic for new discussions.